lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -247,13 +253,15 @@ static int z_to_nrf_flags(int z_flags)
 		nrf_flags |= NRF_MSG_PEEK;
 	}
 
+#if defined (CONFIG_NET_SOCKETS_POSIX_NAMES)
```

Comment:
We should rather add these flags to an appropriate POSIX header in sdk-zephyr instead of disabling them.

Reply:
Yes, agree, but that would be a zephyr change. So, until that, for this PR, we need to go with POSIX_API flagging here?

Reply:
You could post a PR towards `sdk-zephyr` and update manifest in this PR, we already have quite a few downstream changes in the networking headers:
https://github.com/nrfconnect/sdk-zephyr/commit/38a90a7768511cbe769d04ce827e160d8fa554b8
Similar changes could be applied to POSIX headers (symbols unsupported upstream).

Reply:
OK, created PR to sdk-zephyr for adding those sckt flags to posix socket header: https://github.com/nrfconnect/sdk-zephyr/pull/476

I'll update the manifest into this PR when that is merged.

Reply:
Thanks, just one more request, please update the `sdk-zephyr` revision to `pull/476/head` already in the manifest, we want a CI run over that PR.

Reply:
Thanks @rlubos. All should be now done in both related PRes.